model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,51 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - id: platform
+        requirement_level: "required"
+        brief: >
+            This attribute indicates the operating system name of the device on which the crash event occurred.
+        type:
+          allow_custom_values: false
+          members:
+            - id: android
+              value: 'android'
+              brief: 'Value if crash happened on an android device.'
+            - id: ios
+              value: 'ios'
+              brief: 'Value if crash happened on an ios device.'
+      - id: foreground
+        requirement_level: "required"
+        brief: >
+          This attribute indicates whether the app was in the foreground or not during the crash.
+        type: boolean
+      - id: stacktrace
+        requirement_level: recommended
+        brief: >
+          The contents of printStackTrace() being called on the throwable
+          For android crashes
+        type: string
+      - id: thread
+        requirement_level: recommended
+        brief: >
+          A string representation of this thread. Including the thread's name, priority, and thread group
+          For android crashes
+        type: string
+      - id: throwableMessage
+        requirement_level: recommended
+        brief: >
+          The message within the throwable object
+          For android crashes
+        type: string
+      - id: crashReport
+        requirement_level: recommended
+        brief: >
+          The contents from PLCrashReporter
```

Comment:
What is PLCrashReporter? Is that iOS or Android specific or generic?
Also, the brief seems to be not complete here? `For ios crashes ...`